modules/snooker/src/main/ts/ephox/snooker/api/TableFill.ts

```diff
@@ -55,6 +63,16 @@ const cloneFormats = function (oldCell: SugarElement, newCell: SugarElement, for
   }).getOr(newCell);
 };
 
+const transferAppropriateAttributes = <T extends HTMLElement> (original: T, clone: T): void => {
+  Arr.each(Obj.keys(transferableAttributes), (attributeName: string) => {
+    const attribute = original.getAttribute(attributeName);
```

Comment:
Can you use Attribute from Sugar for the get and set parts?

Reply:
As Simon said, please use `Attribute` from sugar here instead. That will also allow this to be cleaned up a little more.